ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsService.scala

```diff
@@ -272,27 +272,36 @@ class ContractsService(
   ): Source[Error \/ domain.ActiveContract[LfValue], NotUsed] =
     searchInMemory(jwt, party, Set(templateId), queryParams)
 
+  @SuppressWarnings(Array("org.wartremover.warts.Any"))
   private[http] def insertDeleteStepSource(
       jwt: Jwt,
       party: lar.Party,
       templateIds: List[domain.TemplateId.RequiredPkg],
+      startOffset: Option[domain.StartingOffset] = None,
       terminates: Terminates = Terminates.AtLedgerEnd,
   ): Source[ContractStreamStep.LAV1, NotUsed] = {
 
     val txnFilter = util.Transactions.transactionFilterFor(party, templateIds)
-    val source = getActiveContracts(jwt, txnFilter, true)
+    def source = getActiveContracts(jwt, txnFilter, true)
 
     val transactionsSince
       : api.ledger_offset.LedgerOffset => Source[api.transaction.Transaction, NotUsed] =
       getCreatesAndArchivesSince(jwt, txnFilter, _: api.ledger_offset.LedgerOffset, terminates)
 
-    import ContractsFetch.acsFollowingAndBoundary, ContractsFetch.GraphExtensions._
-    val contractsAndBoundary = acsFollowingAndBoundary(transactionsSince).divertToHead
-    source
-      .viaMat(contractsAndBoundary) { (nu, fob) =>
-        fob.foreach(a => logger.debug(s"contracts fetch completed at: ${a.toString}"))
-        nu
-      }
+    // TODO SC use startOffset
```

Comment:
why TODO? you are already sourcing the `startOffset` into `transactionsFollowingBoundary` flow.

Reply:
Just forgot to delete it :)